chromium_src/chrome/install_static/install_util.cc

```diff
@@ -3,15 +3,46 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this file,
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+#include "chrome/install_static/install_util.h"
+
 #include "chrome/install_static/buildflags.h"
+#include "components/version_info/channel.h"
 
 #if defined(OFFICIAL_BUILD)
 #undef BUILDFLAG_INTERNAL_USE_GOOGLE_UPDATE_INTEGRATION
 #define BUILDFLAG_INTERNAL_USE_GOOGLE_UPDATE_INTEGRATION() (1)
 #endif
 
+#define GetChromeChannel GetChromeChannel_Unused
+
 #include "../../../../chrome/install_static/install_util.cc"
 
+#undef GetChromeChannel
+
 #if defined(OFFICIAL_BUILD)
 #undef BUILDFLAG_INTERNAL_USE_GOOGLE_UPDATE_INTEGRATION
 #endif
+
+namespace install_static {
+
+version_info::Channel GetChromeChannel() {
```

Comment:
I think it might be cleaner to redefine as GetChromeChannel_ChromiumImpl and then in our version just check `channel_name == L"nightly"` and then fall back onto `GetChromeChannel_ChromiumImpl`, WDYT?

Reply:
There's another difference here: the original code only does things if `GOOGLE_UPDATE_INTEGRATION` is defined, and we want to do that if `OFFICIAL_BUILD` is defined instead. However, we're already temporarily setting `GOOGLE_UPDATE_INTEGRATION` above anyway, so I agree with you that's better (and more future-proof).

Fixed. Thanks for pointing this out.